scripts/install.sh

```diff
@@ -111,10 +77,5 @@ else
 fi
 }
 
-
-
-install_bluemix_cli
-bluemix_auth
-cluster_setup
 initial_setup
 health_check
```

Comment:
I don't think we need either of these functions? Also, there is no health_check?